src/itwinai/torch/trainer.py

```diff
@@ -1220,3 +1231,390 @@ def dist_train(
                 dist.destroy_process_group()
 
     return dist_train
+
+
+DEFAULT_RAY_CONFIG = {
```

Comment:
In general, I dislike the idea of having constants like this in the middle of a file. This clutters the file a bit and makes it harder to maintain in the future. I think I would prefer either 

1) putting it in a different file (`constants.py` or something) or 

2) at least putting it at the top of this file.

Reply:
Okay I'll move it to the top, but keep in mind this is a temporary fix before I figure out how to handle the config better 😌